app/src/main/java/org/fossasia/pslab/fragment/LuxMeterFragmentConfig.java

```diff
@@ -157,8 +162,23 @@ else if (value < updatePeriodMin)
             @Override
             public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
                 try {
-                    if (bh1750 != null) {
-                        bh1750.setRange(gainValue.getSelectedItem().toString());
+                    switch (position) {
+                        case 0:
+                        case 1:
+                        case 2:
+                            if (bh1750 != null) {
+                                bh1750.setRange(gainValue.getSelectedItem().toString());
+                            }
+                            break;
+                        case 3:
+                        case 4:
+                        case 5:
```

Comment:
@Vikum94 I think you can combine switch cases in single line eg. case 1: case 2: case 3:

Reply:
@abhinavraj23 When file reformatted the result is this. Do you want me to change it?